src/psyclone/psyGen.py

```diff
@@ -3124,26 +3117,24 @@ def __str__(self):
         result += "EndLoop"
         return result
 
-    def has_inc_arg(self, mapping={}):
+    def has_inc_arg(self):
         ''' Returns True if any of the Kernels called within this
         loop have an argument with INC access. Returns False otherwise '''
-        assert mapping != {}, "psyGen:Loop:has_inc_arg: Error - a mapping "\
-            "must be provided"
         for kern_call in self.kern_calls():
             for arg in kern_call.arguments.args:
-                if arg.access.lower() == mapping["inc"]:
+                if arg.access == AccessType.INC:
                     return True
         return False
 
-    def unique_modified_args(self, mapping, arg_type):
+    def unique_modified_args(self, arg_type):
```

Comment:
Please document the interface to this routine.

Reply:
The risk of "you touch it, you document it", since I don't really know the dynamo API - sorry, I've applied your changes.

Reply:
Ah, I see. I hadn't thought of that drawback before. Mind you, the chances are that nobody else is much the wiser either at this point (since if it hasn't got an interface description then we haven't touched it in quite a while).